Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -153,6 +145,18 @@ - (instancetype)initWithApp:(FIRApp *)app
                                                          settings:settings
                                                     onDemandModel:onDemandModel];
 
+    if (sessions) {
```

Comment:
If there is a significance in delaying this initialization, would recommend comments qualifying why delayed initialization is necessary.

Reply:
I put the comments down where I call `registerWithSubscriber`